src/_guides/language/language-tour.md

```diff
@@ -3875,7 +3897,7 @@ class Foo<T [!extends SomeBaseClass!]> {
 class Extender extends SomeBaseClass {...}
 {% endprettify %}
 
-It's OK to use `SomeBaseClass` or any of its subclasses as generic argument:
+It's OK to use `SomeBaseClass` or any of its subclasses as the generic argument:
```

Comment:
Not sure if it's worth going into, but it's not just subclasses - it's subtypes in general (i.e. also things that implement SomeBaseClass).